gettsim/transfers/rente.py

```diff
@@ -218,32 +196,35 @@ def entgeltp_update_lohn(
     """
 
     # Scale bruttolohn up if earned in eastern Germany
-    bruttolohn_scaled_east = bruttolohn_m
-    bruttolohn_scaled_east.loc[wohnort_ost] = (
-        bruttolohn_scaled_east.loc[wohnort_ost]
-        * ges_rente_params["umrechnung_entgeltp_beitrittsgebiet"]
-    )
+    if wohnort_ost:
```

Comment:
Oh, maybe make a note/issue: I thought this parameters only applies to what was earned during GDR times, now current stuff ?